sublime-package.json

```diff
@@ -0,0 +1,77 @@
+                                    "additionalProperties": false,
+                                    "properties": {
+                                        "trace.server": {
+                                            "type": "boolean",
+                                            "markdownDescription": "Enable/disable trace logging of client and server communication"
+                                        },
+                                        "elmPath": {
+                                            "type": "string",
+                                            "markdownDescription": "The path to your `elm` executable. Should be empty by default, in that case it will assume the name and try to first get it from a local npm installation or a global one. If you set it manually it will not try to load from the npm folder."
+                                        },
+                                        "elmFormatPath": {
+                                            "type": "string",
+                                            "markdownDescription": "The path to your `elm-format` executable. Should be empty by default, in that case it will assume the name and try to first get it from a local npm installation or a global one. If you set it manually it will not try to load from the npm folder."
+                                        },
+                                        "elmTestPath": {
+                                            "type": "string",
+                                            "markdownDescription": "The path to your elm-test executable. Should be empty by default, in that case it will assume the name and try to first get it from a local npm installation or a global one. If you set it manually it will not try to load from the npm folder."
+                                        },
+                                        "elmAnalyseTrigger": {
+                                            "type": "string",
+                                            "default": "never",
+                                            "enum": ["change", "save", "never"],
+                                            "markdownDescription": "elmLS.elmAnalyseTrigger: `elm-analyse` executed on `'change'`, `'save'` or `'never'` (default: `'never'`)"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "type": "object",
+                    "allOf": [
+                        {
+                            "$ref": "sublime://settings/LSP-plugin-base"
+                        },
+                        {
+                            "$ref": "sublime://settings/LSP-elm#/definitions/PluginConfig"
+                        }
+                    ]
+                }
+            },
+            {
+              "file_patterns": [
```

Comment:
You have some inconsistent indention (sometimes 2, sometimes 4 spaces).

You can use `LSP Format Document` to fix that.

Reply:
ab5545c